app/controllers/admin/tags/moderators_controller.rb

```diff
@@ -0,0 +1,44 @@
+module Admin
+  module Tags
+    class ModeratorsController < Admin::ApplicationController
+      def authorization_resource
+        Tag
+      end
```

Comment:
This is to monkey patch `ApplicationController.authorization_resource` for the related `InternalPolicy` (which should be renamed to `AdminPolicy` in a future PR). Without it, `authorization_resource` returns `Moderator`.